api_tests/lib_sdk/cnd_instance.ts

```diff
@@ -39,27 +38,20 @@ export class CndInstance {
             "config.toml"
         );
 
-        const pemObject = new PEMObject("SEED", this.actorConfig.seed);
```

Comment:
Now that this is removed, we can no longer control the seed of the created nodes. It seems be fine because the tests pass but it means we should probably remove the seed from the E2EActorConfig class :)